apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ast/AstBuilder.kt

```diff
@@ -353,14 +347,13 @@ internal class AstBuilder private constructor(
   ): CodeGenerationAst.ObjectType {
     val schemaType = schema.resolveType(schema.resolveType(this.type.rawType.name!!))
     return buildObjectType(
-        name = responseName,
+        name = normalizedTypeName,
         description = schemaType.description,
         schemaTypename = this.type.rawType.name,
         fields = fields,
         fragments = fragments,
         targetPackageName = targetPackageName,
         abstract = abstract,
-
```

Comment:
Thanks for removing these, I think they came from a previous refactoring of mine.